tools/cldr/gentzinfo.js

```diff
@@ -218,4 +219,50 @@ fs.writeFile(filepath, JSON.stringify(countryToZones, true, 4), function (err) {
 	}
 });
 
+// generate timezone.jf
+var rootTimeZoneID = {"timezone": "Etc/UTC"};
+fs.writeFileSync(path.join(toDir, "timezone.jf"), JSON.stringify(rootTimeZoneID, true, 4), "utf-8");
+
+var defaultTimeZoneID = {
+    "AU": "Australia/Sydney",
+    "BD": "Asia/Dhaka",
+    "BR": "America/Sao_Paulo",
+    "CA": "America/Toronto",
+    "CL": "America/Santiago",
+    "CN": "Asia/Shanghai",
+    "CY": "Asia/Nicosia",
+    "ES": "Europe/Madrid",
+    "FM": "Pacific/Chuuk",
+    "GB": "Europe/London",
+    "GL": "America/Nuuk",
+    "IE": "Europe/Dublin",
+    "IN": "Asia/Kolkata",
+    "KE": "Africa/Nairobi",
+    "KI": "Pacific/Kiritimati",
+    "MH": "Pacific/Majuro",
+    "MM": "Asia/Yangon",
+    "MN": "Asia/Ulaanbaatar",
+    "MO": "Asia/Macau",
+    "MX": "America/Mexico_City",
+    "NO": "Europe/Oslo",
+    "NZ": "Pacific/Auckland",
+    "PF": "Pacific/Tahiti",
+    "PG": "Pacific/Port_Moresby",
+    "PT": "Europe/Lisbon",
+    "RU": "Europe/Moscow",
+    "US": "America/New_York"
+}
+
+for (var country in countryToZones) {
+    var directory = path.join(toDir, "und", country);
+    if (!fs.existsSync(directory)) {
+        mkdirs(directory);
+    };
+    var data = {
+        timezone: typeof defaultTimeZoneID[country] !== "undefined" ? defaultTimeZoneID[country]: countryToZones[country][0]
```

Comment:
Here is where we need to read in the tzdata file called "backward" and map from the LINK-NAME column to the TARGET column.